<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Order</class>
 <widget class="QDialog" name="Order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>96</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Customer Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="product">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Product</string>
   </property>
  </widget>
  <widget class="QLabel" name="quantity">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>190</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Qty</string>
   </property>
  </widget>
  <widget class="QLabel" name="rate">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>190</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="amount">
   <property name="geometry">
    <rect>
     <x>725</x>
     <y>190</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="grossamount">
   <property name="geometry">
    <rect>
     <x>713</x>
     <y>289</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gross Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="vat">
   <property name="geometry">
    <rect>
     <x>713</x>
     <y>319</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>VAT 13%</string>
   </property>
  </widget>
  <widget class="QLabel" name="discount">
   <property name="geometry">
    <rect>
     <x>713</x>
     <y>349</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount</string>
   </property>
  </widget>
  <widget class="QLabel" name="nettotal">
   <property name="geometry">
    <rect>
     <x>713</x>
     <y>382</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Net Amount</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_user">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>53</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_prod">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_qty">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>220</y>
     <width>54</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_discount">
   <property name="geometry">
    <rect>
     <x>828</x>
     <y>349</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_amount">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_gross">
   <property name="geometry">
    <rect>
     <x>828</x>
     <y>287</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_vat">
   <property name="geometry">
    <rect>
     <x>828</x>
     <y>321</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_total">
   <property name="geometry">
    <rect>
     <x>828</x>
     <y>380</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_phone">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>94</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_rate">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="create_order">
   <property name="geometry">
    <rect>
     <x>389</x>
     <y>440</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Order</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_order">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Proceed_Cal">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>430</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Proceed Transaction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sales_order">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Sales Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_vinv">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>View Inventory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_account">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="report">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Info History</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
